graylog2-web-interface/test/helpers/mocking/StoreProviderMock.js

```diff
@@ -0,0 +1,14 @@
+export default class StoreProviderMock {
+  constructor(stores = {}) {
+    this.stores = stores;
+  }
+
+  getStore(name) {
+    const result = this.stores[name] || {};
+    if (!result[`${name}Store`]) {
+      result[`${name}Store`] = {};
```

Comment:
In order to make this consistent with `CombinedProviderMock`, it should return a configurable default store mock when there is no store registered for it.